public/scss/_forms.scss

```diff
@@ -73,6 +73,12 @@ form.cra-form {
   input,
   select {
     @extend %form-element;
+    width: 300px;
+
```

Comment:
how did this sneak in here 🤔 
should we maybe consider a percentage value, and then set max-width in pixels? not a blocker, just something to think about

Reply:
removed this stuff, I was trying to get the label larger because I didn't want it spilling on to 3 lines but it's really not that bad

Reply:
also didn't realize it was making some inputs larger